From: Lucky Mahendra Purba <luckymahendra080503@gmail.com>
Date: Fri, 23 Jan 2026 13:41:10 +0000
Subject: [PATCH] <short summary of the patch>

TODO: Put a short summary on the line above and replace this paragraph
with a longer explanation of this change. Complete the meta-information
with other relevant fields (see below for details). To make it easier, the
information below has been extracted from the changelog. Adjust it or drop
it.
---
 helpers/calamares-sources-final | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helpers/calamares-sources-final b/helpers/calamares-sources-final
index 8eff21a..242e019 100755
--- a/helpers/calamares-sources-final
+++ b/helpers/calamares-sources-final
@@ -6,7 +6,7 @@
 CHROOT=$(mount | grep proc | grep calamares | awk '{print $3}' | sed -e "s#/proc##g")
 RELEASE="verbeek"
 
-cat <<EOF > $CHROOT/etc/apt/blankon.sources
+cat <<EOF > $CHROOT/etc/apt/sources.list.d/blankon.sources
 Types: deb deb-src
 URIs: http://arsip-dev.blankonlinux.id/dev/
 Suites: $RELEASE
